modules/oadp-self-service-admin-spec-enforce-nab.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// backup_and_restore/application_backup_and_restore/oadp-self-service/oadp-self-service-cluster-admin-use-cases.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-self-service-admin-spec-enforce-nab_{context}"]
+= Self-Service administrator spec enforcement for NAB
+
+As a cluster administrator, you can enforce the following fields for a `NonAdminBackup` (NAB) CR:
```

Comment:
IMO we should change the approach here, we should document what spec fields are non-enforceable by the admins and the special cases etc. Velero API keeps evolving per release, documenting allowlist is a difficult task.

Reply:
@shubham-pampattiwar wdyt about us creating a kbase article that we can update and perhaps that is something the docs could point to.  I'm not 100% sure the docs team is allowed to point at a kbase, it's worth asking.  If we do think it's going to be very dynamic release to release I would suggest we exclude it from the production docs and ONLY have a kbase article.